HW05-byteCodes/src/main/java/ru/mts/MyIOC.java

```diff
@@ -0,0 +1,39 @@
+package ru.mts;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.InvocationHandler;
+import java.lang.reflect.Method;
+import java.lang.reflect.Proxy;
+
+public class MyIOC {
+
+    public MyIOC() {
+    }
+
+    static TestLoggingInterface createTestLogging() throws Exception {
+        TestLogging myObject = new TestLogging();
+        InvocationHandler handler = new MyInvocationHandler(myObject);
+        return (TestLoggingInterface) Proxy.newProxyInstance(myObject.getClass().getClassLoader(), myObject.getClass().getInterfaces(), handler);
+
+    }
+
+    static class MyInvocationHandler implements InvocationHandler {
+        private final TestLoggingInterface TestLogging;
```

Comment:
```suggestion
        private final TestLoggingInterface testLogging;
```

Поля принятоПисатьвCamelCase с маленькой буквы